### PR TITLE
Change gci project pool (affecting m59 jobs)

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign @wonderfly

### CODE REVIEW

**Comment by krzyzacy** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/4595/pull-test-infra-bazel/7659/

@ixdy ^^ :confused:

**Comment by wonderfly** [comment]
Suggest rewording the PR title to include `for gci 59 jobs` to make it clear when we look at the commit history.

/lgtm

**Comment by ixdy** [comment]
wat

**Comment by krzyzacy** [comment]
(seems just a bazel flake?)
